src/jdmtool/featunlk.py

```diff
@@ -402,31 +412,70 @@ def display_content_of_dat_file(dat_file: pathlib.Path):
         print('** Revision: ' + chr(header_bytes[0x92]))
         (cycle, f_month, f_day, f_year, t_month, t_day, t_year) = struct.unpack('<HBBHBBH', header_bytes[0x81:0x81+0xa])
         print('** Cycle: ', cycle)
-        print(f'** Effective: {f_year}-{f_month:02}-{f_day:02} to {t_year}-{t_month:02}-{t_day:02}')
+        cus_date1 = (datetime.strptime(f'{f_day:02}{f_month:02}{f_year}', "%d%m%Y").date()).strftime("%d-%b-%Y")
+        cus_date2 = (datetime.strptime(f'{t_day:02}{t_month:02}{t_year}', "%d%m%Y").date()).strftime("%d-%b-%Y")
```

Comment:
If you already have the integer values, you don't need `strptime`; just do e.g. `datetime.date(f_year, f_month, f_day)`.

Also, the `"%d-%b-%Y"` is mixing a locale-specific name (`%b`) and a fixed dd-mm-yyyy format. You should pick either a "neutral" format like `%Y-%m-%d`, or a fully locale-specific like `%x`.

Reply:
Oh ok, sounds good